### PR TITLE
Reorganize FAQ and add several FAQ entries

### PR DESCRIPTION
**Pull request by adrian-lara**
With users testing the upcoming SEED release of 2.7.4 (from 2.6.2-patchxx), there has been an influx of questions from specific users. We thought it would be useful to collect and generalize some of these on this site. 

To do that, we thought it would be beneficial to reorganize the FAQ. This PR does that as well as adds a few questions and answers that we've answered recently.

Screen Shot 2020-10-15 at 10 07 00 AM

Looking for any feedback, but particularly wondering others' thoughts on the use of sub-sections, the actual sub-sections used, and the relevance of the Qs and As in each sub-section.

### CODE REVIEW

File: docs/faq/features/inventory_management.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# FAQ - Features - Inventory Management
```
Comment (adrian-lara) [inline docs/faq/features/inventory_management.md:1]: Qs and As are new - content should be reviewed.

**Comment by RDmitchell** [comment]
@adrian-lara / @axelstudios / @nllong -- I have said this before and maybe we can do this in November -- I think the website needs to be reorganized, so that there is a clear demarcation between documentation for developers / web admins, and users. 

Right now it is all lumped together, and one person I talked to that was a potential user, not a developer, said he completely misunderstood what SEED was about as a user, because the Getting Started section's main information was for developers. 

For example, if we break the website at the top level into For Users and For Developers / Web Admin, then there would be a separate FAQ under each section.

**Comment by nllong** [comment]
I totally agree. Is this something that you would like to tackle? Everything is in markdown.

Da: RDmitchell <notifications@github.com>
Data: giovedì, 15 ottobre 2020 10:50
A: SEED-platform/website <website@noreply.github.com>
Cc: Long, Nicholas <Nicholas.Long@nrel.gov>, Mention <mention@noreply.github.com>
Oggetto: Re: [SEED-platform/website] Reorganize FAQ and add several FAQ entries (#2)

@adrian-lara<https://gcc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fadrian-lara&data=04%7C01%7Cnicholas.long%40nrel.gov%7C0ba2c446964b4084be8a08d8712a5bd6%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637383774026094359%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=S56STifXKDJvqdLIoMPFE8pS5184BjF0dKE1ZTg9hu0%3D&reserved=0> / @axelstudios<https://gcc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Faxelstudios&data=04%7C01%7Cnicholas.long%40nrel.gov%7C0ba2c446964b4084be8a08d8712a5bd6%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637383774026104312%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=naRNnzHtnMKql%2BVJai%2Bgp%2B4XtnjbOEylKxTKzKxc5PI%3D&reserved=0> / @nllong<https://gcc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnllong&data=04%7C01%7Cnicholas.long%40nrel.gov%7C0ba2c446964b4084be8a08d8712a5bd6%7Ca0f29d7e28cd4f5484427885aee7c080%7C0%7C0%7C637383774026104312%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=uweMyWgzNa%2Fc4sSEMCCqOOVgict4o72wOsiX3WTj1BU%3D&reserved=0> -- I have said this before and maybe we can do this in November -- I think the website needs to be reorganized, so that there is a clear demarcation between documentation for developers / web admins, and users.

Right now it is all lumped together, and one person I talked to that was a potential user, not a developer, said he completely misunderstood what SEED was about as a user, because the Getting Started section's main information was for developers.

For example, if we break the website at the top level into For Users and For Developers / Web Admin, then there would be a separate FAQ under each section.

**Comment by RDmitchell** [comment]
@nllong -- yes, it's on my "list of things to do" !! I am thinking I would come up with a design for reorganizing it, and send that around for comment, before actually making any changes. Hopefully I will get to this in November. I'm still working on closing out 2020 projects in October  :<

File: docs/faq/features/geocoding.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+1. Your organization has a MapQuest API key. You can register for one through [MapQuest's website](https://developer.mapquest.com/plan_purchase/steps/business_edition/business_edition_free/register) and apply it on your organization settings page.
+2. The records being geocoded have address values that can be read by SEED. In your organization column settings page, you can specify which fields and in what order SEED will use to build the full address that will be geocoded by MapQuest.
+3. The records being geocoded _do not_ already have latitude and longitude populated. SEED won't override these values, but you can edit and remove these values if you want SEED to attempt to generate them with MapQuest.
+
+SEED will make this attempt in the following cases:
+
+* During the file import process, after you've mapped columns, SEED will automatically attempt geocoding on records.
+
+* On either the properties page or the tax lots page, you can select records and "Geocode Selected".
```
Comment (kflemin) [inline docs/faq/features/geocoding.md:15]: is "Geocode Selected" in an action menu or a link on the page?  (maybe mention when the button is)

File: docs/faq/features/geocoding.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Note: Valid UBID (properties) or ULID (tax lots) values provide a latitude and longitude, and these can be used to provide latitude and longitude values. On import, UBID/ULID is used instead of MapQuest if available. On the inventory pages, there's a separate button to Decode UBID/ULID for Selected.
+
+## I just geocoded my data using SEED. How do I review and verify the results?
+
+There are multiple ways to do this. The most obvious way is to verify that latitude and longitude fields are populated. Additionally, SEED provides a field called "Geocoding Confidence". This field can provide evidence of how the geocoding went for that record - a value starting with "Low..." indicates that geocoding failed for a record because the address values didn't yield reliable results. Finally, SEED provides an inventory map page that allows you to view where the points are located on a map.
```
Comment (kflemin) [inline docs/faq/features/geocoding.md:21]: "There are multiple ways of verifying geocoding results."

File: docs/faq/features/inventory_management.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Pairing, Merging, Linking, and Matching - What do each of those mean and how are they related or different from each other?
+
+Different inventory management and analysis features in SEED are based on the relationships between records of different inventory types (properties and tax lots). Those terms help describe these relationships.
+
+* Pairing refers to the association between properties and tax lots within the same cycle.
+* Merging refers to the act of combining multiple properties (or multiple tax lots) into one record. This can be done manually by users or automatically by SEED and helps avoid deduplication of data.
```
Comment (kflemin) [inline docs/faq/features/inventory_management.md:8]: deduplication?

File: docs/faq/getting_help.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# FAQ - Getting Help
+
+## What resources or support do you provide to jurisdictions interested in using SEED as part of a benchmarking program?
+
+We typically provide technical support within reason as needed. As the community grows, there are more users able to provide support. It is recommended to use [GitHub issues](https://github.com/SEED-platform/seed/issues) to a) view existing feature requests and issues, and b) communicate with various developers.
```
Comment (kflemin) [inline docs/faq/getting_help.md:5]: "We typically provide as needed technical support within reason."

**Review by kflemin — APPROVED** [review]
looks good!  just a couple of comments...

File: docs/faq/features/inventory_management.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+* Linking refers to the association between properties across cycles (or tax lots across cycles) and is useful for analysis.  
+* Matching refers to whether two or more properties match (or two or more tax lots). A match can only happen if specific fields between records match. Records can be compared within the same cycle (triggering a merge) or across cycles (building a link). The specific fields can be modified for each organization.
+
+For more details, refer to [this documentation that covers matching, merging, and linking](https://github.com/SEED-platform/seed/blob/develop/docs/source/matching.rst).
+
+
+## The linking feature is new - can I get more details on how it works?
+
+Cross-cycle linking, or simply **linking**, allows organizations to analyze uploads of the same property (or tax lot) over time. This is done by matching records in one cycle with records within other cycles. SEED builds these associations automatically during import using matching criteria as specified by the organization. Additionally, this association can be made during a hand-edit to record values (especially if the changed value is a matching criteria field).
```
Comment (kflemin) [inline docs/faq/features/inventory_management.md:17]: manual edit instead of "hand-edit".  @nllong

File: docs/faq/getting_help.md (hunk 5 above)
Comment (adrian-lara) [reply docs/faq/getting_help.md]: Went with
> We typically provide technical support as needed and within reason.
Hope that's OK.

Comment (kflemin) [reply docs/faq/getting_help.md]: 👍

**Comment by adrian-lara** [comment]
Thanks @kflemin ! I made changes based on your feedback

**Review by kflemin — APPROVED** [review]

**Comment by adrian-lara** [comment]
Per discussion, Nick is good with merging this without review. He mentioned more changes might be coming as well - just as a note.

**Pull request merged**